Makefile

```diff
@@ -69,8 +69,8 @@ endif
 # weird reports at golangci-lint step
 ifneq ($(HAS_SERVER),)
 	@echo Running golangci-lint
-	$(GO) vet ./server/...
-	$(GOBIN)/golangci-lint run ./server/...
+	$(GO) vet ./server/... ./build/...
```

Comment:
Do we still need this? Or will `$(GO) vet ./...` work since we changed how the monorepo checkout works?

Reply:
The reason I ask, is we want to keep the `Makefile` as identical to the starter template's `Makefile` as we can. This change creates drift between the two repos

Reply:
@mickmister Yeah we still require the file-hound package and its respective files.